app/schemas/models/user.coffee

```diff
@@ -55,8 +55,8 @@ _.extend UserSchema.properties,
   email: c.shortString({title: 'Email', format: 'email'})
   emailVerified: { type: 'boolean' }
   iosIdentifierForVendor: c.shortString({format: 'hidden'})
-  firstName: c.shortString({title: 'First Name'})
-  lastName: c.shortString({title: 'Last Name'})
+  firstName: c.shortString({title: 'First Name', not: {pattern: 'Q204384420'}})
```

Comment:
This will fail all operations on a user object that has already been created with this pattern.  I would assume this is acceptable in this case but want to call out.  Alternatively we can simply include these lines in the modal schema.

Reply:
Yes, these are bogus users, they can error to their heart's content if anyone ever logs into one of them (unlikely).